content/token-swap.md

```diff
@@ -150,10 +155,14 @@ transaction.add(createTokenAccountPoolInstruction)
 transaction.add(initializeTokenAccountPoolInstruction)
 ```
 
+JC: feel free to use the creator's associated token account here!
+
 ### Pool Token Fee Account
 
 The pool token fee account is the token account that the fees for the token swaps are paid to. This account must be owned by a specific account defined in the swap program - that account has public key [HfoTxFR1Tm6kGmWgYWD6J7YHVy1UwqSULUGVLXkJqaKN](https://explorer.solana.com/address/HfoTxFR1Tm6kGmWgYWD6J7YHVy1UwqSULUGVLXkJqaKN?cluster=devnet).
 
+JC: if we encourage people to use their own program (or deploy or own), then it doesn't need to be owned by that address, it's a holdover from when Serum was running token-swap.
```

Comment:
@joncinque The account address would be defined by whatever swap program they're using, correct? Or is there some deterministic way to know the account based on the token swap program's address?

Reply:
It *can* be, if the program has constraints defined.  The previous deployment forced the fee owner to be a particular address, but that isn't the default behavior.  You can read more about constraints at https://github.com/solana-labs/solana-program-library/tree/master/token-swap#building -- either way, this is *not* typically deterministic

Reply:
Got it